code/routers/user/profile.js

```diff
@@ -65,7 +65,7 @@ router.get('/profile', (req, res) => {
               });
             }
             else{
-              userData['donationCount'] = user[0].count;
+              userData['donationCount'] = user[0] ? user[0].count : 0;
```

Comment:
This is already on master

Reply:
It should be fine then. It shouldn't cause any problems, but good to know! thanks!